Doc/library/typing.rst

```diff
@@ -604,18 +604,16 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
    * Redundant arguments are skipped, e.g.::
 
-       Union[int, str, int] == Union[int, str]
+       Union[int, str, int] == Union[int, str] == int | str
 
    * When comparing unions, the argument order is ignored, e.g.::
 
        Union[int, str] == Union[str, int]
 
-   * You cannot subclass or instantiate a union.
+   * You cannot subclass or instantiate a ``Union``.
 
    * You cannot write ``Union[X][Y]``.
```

Comment:
Oh I just noticed this and maybe this can be another PR (since it needs to be backported to 3.9) but isn't this inaccurate if X is a `TypeVar`?

Reply:
At least the sentence doesn't sound super relevant to me. All examples show how to use `Union`, so would someone really want to write `Union[X][Y]`? Maybe this is some historic discussion.